sk/src/app.pcss

```diff
@@ -4,67 +4,48 @@
  
 @layer base {
   :root {
-    --background: 0 0% 100%;
-    --foreground: 240 10% 3.9%;
- 
```

Comment:
make the spacing the same at least, easier to diff :p

Reply:
fixed spacing